Surface_mesh/doc/Surface_mesh/Surface_mesh.txt

```diff
@@ -81,7 +81,7 @@ The following example shows how to create a very simple `Surface_mesh`
 by adding 2 faces, and how to check that a face is correctly added
 to the mesh.
 
-\cgalExample{Surface_mesh/sm_check_wrong_orientation.cpp}
+\cgalExample{Surface_mesh/check_orientation.cpp}
```

Comment:
You must also rename `examples/Surface_mesh/sm_check_wrong_orientation.cpp` itself, in Git.

Reply:
? It is done, I made a git mv